### PR TITLE
feat: Add classNames and styles prop for card

### LINKED ISSUE #46260: Add className for inner child div's of components (i.e Card)
### What problem does this feature solve?

For a better usability with TailwindCSS, I suggest that components that has one inner child that wraps everything that it'll be easy to add className, for example the body of a Card.
Moreover I think antd can improve in manner of customizations, lot's of properties are still hidden even in v5.
If tailwindcss can be used with it, it'l be great.
sometime there is a need to use important over antd classes which I didn't still realize why it rise above the immediate className being applied.

Thank you!

### What does the proposed API look like?

`<Card className="items-strech"/>` 
Or if you don't to use the className prop maybe create something similar to the `bodyStyle`

### PR DESCRIPTION
**Pull request by zh-lx**
A new PR from #46276. close #46260 

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

#46260 

### 💡 Background and solution

Add classNames and styles prop for card

### 📝 Changelog

Add classNames and styles prop for card

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Add classNames and styles prop for card      |
| 🇨🇳 Chinese |    Card 组件增加 classNames 和 styles 属性       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/card/Card.tsx [hunk 1]
```diff
@@ -89,15 +113,36 @@ const Card = React.forwardRef<HTMLDivElement, CardProps>((props, ref) => {
...
+  if (process.env.NODE_ENV !== 'production') {
+    const warning = devUseWarning('Card');
+    [
+      ['headStyle', 'styles.header'],
+      ['bodyStyle', 'styles.body'],
+    ].forEach(([deprecatedName, newName]) => {
+      warning.deprecated(!(deprecatedName in props), deprecatedName, newName);
+    });
+  }
+
```
Comment (Wxh16144) [inline components/card/Card.tsx:133]: 弃用消息在 interface 处用 jsDoc 注释一下。参考这里 https://github.com/ant-design/ant-design/blob/9ece76b269f7608cbdf4323a660f99dd3ae10242/components/modal/interface.ts#L58-L60

Comment (zh-lx) [reply components/card/Card.tsx:135]: done

**Comment by zh-lx** [comment]
I add a new table for `styles and classNames` in markdown of Card. And need you change `3` to `4` in [scripts/__snapshots__/check-site.ts.snap#L43-L45](https://github.com/ant-design/ant-design/blob/master/scripts/__snapshots__/check-site.ts.snap#L43-L45) @Wxh16144 @MadCcc

**Comment by MadCcc** [comment]
You can update snapshot now, we've disabled bot.

**Comment by zh-lx** [comment]
> You can update snapshot now, we've disabled bot.

done❤

**Comment by afc163** [comment]
@MadCcc ping~

**Comment by zh-lx** [comment]
@MadCcc Ping一下大佬

**Comment by afc163** [comment]
冲突了

**Comment by zh-lx** [comment]
> 冲突了

resolved~

File: components/card/index.en-US.md [hunk 2]
```diff
@@ -55,6 +53,8 @@ Common props ref：[Common props](/docs/react/common-props)
 | tabProps | [Tabs](/components/tabs/#tabs) | - | - |  |
 | title | Card title | ReactNode | - |  |
 | type | Card style type, can be set to `inner` or not set | string | - |  |
+| classNames | Config Card build-in module's className | Record<SemanticDOM, string> | - | 5.13.0 |
```
Comment (li-jia-nan) [inline components/card/index.en-US.md:56]: 改成 5.14.0 吧

**Review by MadCcc — APPROVED** [review]

**Pull request merged**